src/renderer/components/+catalog/catalog.tsx

```diff
@@ -153,7 +154,7 @@ export class Catalog extends React.Component<Props> {
     if (activeCategory) {
       navigate(catalogURL({ params: {group: activeCategory.spec.group, kind: activeCategory.spec.names.kind }}));
     } else {
-      navigate(catalogURL());
+      navigate(`${catalogURL()}/browse`);
```

Comment:
This url-builder supports parameters: `catalogURL({group: browseCatalogPage})`

Reply:
Yes, it supports parameters but the form is more verbose, I'm not sure if it's fit better:
```
catalogURL({ params: { group: browseCatalogPage }})
```

Reply:
Yes, using your suggested form for sake of consistency.